### PR TITLE
Update local installation guide - checking installation status + troubleshooting

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

- Updated the suggested way to check the installation status
- Updated the "troubleshooting" section
- Minor updates to cmd formatting

**Related issue(s)**
#1921

### CODE REVIEW

File: docs/kyma/docs/030-inst-local-installation-from-release.md [hunk 1]
```diff
@@ -101,16 +101,13 @@ To learn how to test Kyma, see the **Testing Kyma** document.
...
+The `is-installed.sh` script is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.
 
-Run this command to fetch all Pods in all Namespaces:
+If the script indicates the installation failed, try installing Kyma again by re-running the `run.sh`.
 
-  ``` bash
-  kubectl get pods --all-namespaces
-  ```
-The command retrieves all Pods from all Namespaces, the status of the Pods, and their instance numbers. Check if the **STATUS** column shows `Running` for all Pods. If any of the Pods that you require do not start successfully, perform the installation again.
+If the installation fails in a reproducible manner, don't hesitate to reach out on the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
```
Comment (sjanota) [inline docs/kyma/docs/030-inst-local-installation-from-release.md]: 👍

File: docs/kyma/docs/030-inst-local-installation-from-release.md [hunk 2]
```diff
@@ -101,16 +101,13 @@ To learn how to test Kyma, see the **Testing Kyma** document.
...
 Follow the guidelines in the subsections to confirm that your Kubernetes API Server is up and running as expected.
 
-### Access Kyma with CLI
+### Verify the installation status using the is-installed.sh script
 
-Verify the cluster deployment with the kubectl command line interface (CLI).
+The `is-installed.sh` script is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.
 
-Run this command to fetch all Pods in all Namespaces:
+If the script indicates the installation failed, try installing Kyma again by re-running the `run.sh` script.
```
Comment (piotrmsc) [inline docs/kyma/docs/030-inst-local-installation-from-release.md]: Please also mention github issues

Comment (tomekpapiernik) [reply docs/kyma/docs/030-inst-local-installation-from-release.md]: it's done

File: docs/kyma/docs/030-inst-local-installation-from-release.md [hunk 3]
```diff
@@ -101,16 +101,13 @@ To learn how to test Kyma, see the **Testing Kyma** document.
...
```
Comment (klaudiagrz) [inline docs/kyma/docs/030-inst-local-installation-from-release.md]:
```suggestion
If the script indicates that the installation failed, try to install Kyma again by re-running the `run.sh` script.
```

File: docs/kyma/docs/030-inst-local-installation-from-release.md [hunk 4]
```diff
@@ -101,16 +101,13 @@ To learn how to test Kyma, see the **Testing Kyma** document.
...
 
-  ``` bash
-  kubectl get pods --all-namespaces
-  ```
-The command retrieves all Pods from all Namespaces, the status of the Pods, and their instance numbers. Check if the **STATUS** column shows `Running` for all Pods. If any of the Pods that you require do not start successfully, perform the installation again.
+If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out on the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
```
Comment (klaudiagrz) [inline docs/kyma/docs/030-inst-local-installation-from-release.md]:
```suggestion
If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
```

File: docs/kyma/docs/030-inst-local-installation-from-release.md [hunk 5]
```diff
@@ -154,18 +151,26 @@ Kyma uses the autoscaling/v1 stable version, which only provides support for CPU
...
 scripts/is-installed.sh --verbose
 ```
+
+If the installation is successful but a component does not behave in an expected way, see if all deployed Pods are running. Run this command:  
+```
+kubectl get pods --all-namespaces
+```
+
+The command retrieves all Pods from all Namespaces, the status of the Pods, and their instance numbers. Check if the STATUS column shows Running for all Pods. If any of the Pods that you require do not start successfully, perform the installation again.
+If the problem persists, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue or reach out on the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
```
Comment (klaudiagrz) [inline docs/kyma/docs/030-inst-local-installation-from-release.md]:
```suggestion
If the problem persists, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
```

**Pull request merged**